Jenkinsfile

```diff
@@ -135,6 +135,21 @@ try {
 				resourceRequestMemory: '16384Mi'
 			)], serviceAccount: 'bfd') {
 		node(POD_LABEL) {
+			/* This stage switches the gitBranchName (needed for our CCS downsream stages)
+			value if the build is a PR as the BRANCH_NAME var is populated with the build
+			name during PR builds.
+			*/
+			stage('Set Branch Name') {
+				currentStage = env.STAGE_NAME
+				script {
+					if (env.BRANCH_NAME.startsWith('PR')) {
+						gitBranchName = env.CHANGE_BRANCH
+					} else {
+						gitBranchName = env.BRANCH_NAME
+					}
```

Comment:
Obviously, this is just a move of the existing solution from below to here, but I'm curious about Jenkins behavior in this context. I don't think tag references are enabled for the monolithic job anyway, but we're using this logic elsewhere, too:

* When a _tag reference_ is selected, e.g. `17.8.9-test4`, is this true: `env.BRANCH_NAME == '17.8.9-test4'`?
* Do you know what the behavior of `env.TAG_NAME` is when using a non-tag reference, i.e. a branch?

[Available Environment Variables](https://jenkins-east.cloud.cms.gov/bfd/env-vars.html/) for posterity.

Reply:
As discussed, usage of the more specific `env.TAG_NAME` is **not** something we want to consider at this time. `env.BRANCH_NAME` is equal to the tag reference name.